commands/common_args.py

```diff
@@ -0,0 +1,187 @@
+        "--target-job",
+        default=None,
+        multiple=True,
+        help=(
+            "Add a specific target job on which to train on "
+            "(multiple can be passed if --target-job <job_name> is repeated) "
+            "Example: python train.py --target-job BBOX --target-job CLASSIFICATION"
+        ),
+    )
+
+    max_assets = click.option(
+        "--max-assets",
+        default=None,
+        type=int,
+        help="Maximum number of assets to consider",
+    )
+
+    clear_dataset_cache = click.option(
+        "--clear-dataset-cache",
+        default=False,
+        is_flag=True,
+        help="Tells if the dataset cache must be cleared",
+    )
+
+    batch_size = click.option(
+        "--batch-size",
+        default=8,
+        type=int,
+    )
+
+    verbose = click.option("--verbose", default=0, type=int, help="Verbose level")
+
+    randomize_assets = click.option(
+        "--randomize-assets",
+        default=True,
+        type=bool,
+        help="Whether or not to shuffle Kili assets",
+    )
+
+
+def asset_status_in(default: List[AssetStatusT]):
+    default_string = ",".join(default) if default else None
+    return click.option(
+        "--asset-status-in",
+        default=default_string,  # type: ignore
+        callback=lambda _, __, x: x.upper().split(",") if x else None,
+        help=(
+            "Comma separated (without space) list of Kili asset status to select "
+            "among: 'TODO', 'ONGOING', 'LABELED', 'TO_REVIEW', 'REVIEWED'"
+            "Example: python train.py --asset-status-in TO_REVIEW,REVIEWED "
+        ),
+    )
+
+
+class TrainOptions:
+    epochs = click.option(
+        "--epochs",
+        default=50,
+        type=int,
+        show_default=True,
+        help="Number of epochs to train for",
+    )
+    json_args = click.option(
+        "--json-args",
+        default=None,
+        type=str,
+        help=(
+            "Specific parameters to pass to the trainer "
+            "(for example Yolov5 train, Hugging Face transformers, ..."
+        ),
+    )
+
+    disable_wandb = click.option(
+        "--disable-wandb",
+        default=False,
+        is_flag=True,
+        help="Tells if wandb is disabled",
+    )
+
+    asset_status_in = asset_status_in(["LABELED", "TO_REVIEW", "REVIEWED"])
+
+
+class PredictOptions:
+
+    dry_run = click.option(
+        "--dry-run",
+        default=False,
+        is_flag=True,
+        help="Runs the predictions but do not save them into the Kili project",
+    )
+    from_model = click.option(
+        "--from-model",
+        default=None,
+        help="Runs the predictions using a specified model path",
+    )
+    from_project = click.option(
+        "--from-project",
+        default=None,
+        type=str,
+        help=(
+            "Use a model trained of a different project to predict on project_id."
+            "This is usefull if you do not want to pollute the original project with "
```

Comment:
replace by "When a project ID is passed to this argument, it uses a model trained from this project to compute the predictions on the project identified by the --project-id value."